gallery/src/components/NxPagination/NxPaginationExample.tsx

```diff
@@ -10,5 +10,12 @@ import { NxPagination } from '@sonatype/react-shared-components';
 export default function NxPaginationExample() {
   const [page, setPage] = useState(17);
 
-  return <NxPagination onChange={setPage} pageCount={41} currentPage={page} />;
+  return (
+    <div>
```

Comment:
What is the purpose of this `<div>`? The example is already within `<div class="gallery-example-live">` so I don't think that we need to add anymore `<div>`'s

Reply:
I guess I could have used a fragment instead. I just needed to have two components in here.